lms/static/sass/inline-discussion.scss

```diff
@@ -0,0 +1,33 @@
+// From lms-main-v1.scss
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
```

Comment:
@itsjeyd It uses LTR stylesheet, so

1. There should be a corresponding file with RTL + discussions should load the appropriate file (LTR vs RTL).
2. The file should have -ltr suffix, to stay in line with other direction-specific files.